packages/ui/components/ui/tx/view/memo-view.tsx

```diff
@@ -10,8 +10,12 @@ export const MemoViewComponent = ({ memo: { memoView } }: { memo: MemoView }) =>
           <ViewBox
             label='Memo Text'
             visibleContent={
-              // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
-              <div>{memoView.value.plaintext?.text || '&nbsp;'}</div>
+              <div>
+                {/* eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing */}
+                {memoView.value.plaintext?.text || (
+                  <span className='italic text-gray-600'>== none ==</span>
+                )}
+              </div>
```

Comment:
Was confused by seeing `&nbsp;` in the text box. So added a bit of styling instead:

<img width="221" alt="Screenshot 2024-03-20 at 12 29 07 PM" src="https://github.com/penumbra-zone/web/assets/16624263/7e096299-813c-41cd-b5a4-6c58f141775d">